schemas/g-cloud-9.declaration.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Yeah you're right. I guess I was thinking of the framework content getting linked in via bower, that's different though. Just the API does this so far.